cves/CVE-2016-4438.yml

```diff
@@ -144,7 +156,7 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this section
     by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: No interesting commits found
```

Comment:
Do you know what happened in the code between the VCC and the fix?

Reply:
I looked and couldn't really find anything interesting. I  spent a while when for this I was doing research on this one.